packages/capabilities/src/account-usage.js

```diff
@@ -0,0 +1,62 @@
+import { capability, fail, ok, Schema } from '@ucanto/validator'
+import {
+  AccountDID,
+  SpaceDID,
+  equalWith,
+  and,
+  equal,
+  containedWithin,
+} from './utils.js'
+
+/**
+ * Capability can only be delegated (but not invoked) allowing audience to
+ * be derived any `account-usage/` prefixed capability for the account identified
+ * by DID in the `with` field.
+ */
+export const accountUsage = capability({
+  can: 'account-usage/*',
```

Comment:
IDK, maybe we want to do other account things in the future?

```suggestion
  can: 'account/usage/*',
```